feature24/cool-new_testing.txt

```diff
@@ -0,0 +1 @@
+Another new file to test the pull request
```

Comment:
Looks like the testing is going well

Reply:
It seems so...I'm looking forward to some more complicated stuff!